### PR TITLE
Build with base-4.10

### LINKED ISSUE #15: Build failure with GHC HEAD
```
System/Metrics.hs:448:16: error:
    • Constructor ‘Stats.GCStats’ does not have the required strict field(s): mblocksAllocated
    • In the expression:
        Stats.GCStats
          {bytesAllocated = 0, numGcs = 0, maxBytesUsed = 0,
           numByteUsageSamples = 0, cumulativeBytesUsed = 0, bytesCopied = 0,
           currentBytesUsed = 0, currentBytesSlop = 0, maxBytesSlop = 0,
           peakMegabytesAllocated = 0, mutatorCpuSeconds = 0,
           mutatorWallSeconds = 0, gcCpuSeconds = 0, gcWallSeconds = 0,
           cpuSeconds = 0, wallSeconds = 0, parTotBytesCopied = 0,
           parMaxBytesCopied = 0}
      In an equation for ‘emptyGCStats’:
          emptyGCStats
            = Stats.GCStats
                {bytesAllocated = 0, numGcs = 0, maxBytesUsed = 0,
                 numByteUsageSamples = 0, cumulativeBytesUsed = 0, bytesCopied = 0,
                 currentBytesUsed = 0, currentBytesSlop = 0, maxBytesSlop = 0,
                 peakMegabytesAllocated = 0, mutatorCpuSeconds = 0,
                 mutatorWallSeconds = 0, gcCpuSeconds = 0, gcWallSeconds = 0,
                 cpuSeconds = 0, wallSeconds = 0, parTotBytesCopied = 0,
                 parMaxBytesCopied = 0}
```

### PR DESCRIPTION
**Pull request by ocharles**
Fixes #15

### CODE REVIEW

File: System/Metrics.hs [hunk 1]
```diff
@@ -445,6 +445,7 @@ getGcStats = do
 
 -- | Empty GC statistics, as if the application hasn't started yet.
 emptyGCStats :: Stats.GCStats
+#if MIN_VERSION_base(4,10,0)
```
Comment (tibbe) [inline System/Metrics.hs]: When I nest ifdefs I usually indent the nested one one space for readability. Could you change this line to

```
# if MIN_VERSION_base(4,10,0)
```

and similarly for any `else` and `endif`.

**Comment by ocharles** [comment]
Sounds good, done!

**Comment by domenkozar** [comment]
@tibbe hey :) Any chance we could get a release out? It would make testing GHC-8.2.1-rc3 easier

**Comment by RyanGlScott** [comment]
Looks like a new version ([`ekg-core-0.1.1.2`](http://hackage.haskell.org/package/ekg-core-0.1.1.2)) with this change was just uploaded to Hackage.

**Pull request merged**